CONTRIBUTING.md

```diff
@@ -0,0 +1,84 @@
+# Contributing Guidelines
+
+Thanks for interest in contribution to `colin`.
+
+The following is a set of guidelines for contributing to `colin`. These are not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
```

Comment:
I'm puzzled: *...a set of guidelines..* and *...are not rules*. Please make up your mind.